app/assets/stylesheets/common/components/bookmark-modal.scss

```diff
@@ -71,6 +71,7 @@
 
   .bookmark-options-button {
     margin-left: 0.5em;
+    margin-bottom: 9px;
```

Comment:
can we use 1em or 0.5em ? I'm not a big fan of magic number like "9"

Reply:
I think we can, I did it to match the `margin-bottom` of the input that is to it's left. Otherwise it was not centered